Aplib.Core/Belief/BeliefSets/BeliefSet.cs

```diff
@@ -1,14 +1,18 @@
-﻿using Aplib.Core.Belief.Beliefs;
+// This program has been developed by students from the bachelor Computer Science at Utrecht
+// University within the Software Project course.
+// Copyright Utrecht University (Department of Information and Computing Sciences)
+
+using Aplib.Core.Belief.Beliefs;
 using System.Linq;
 
 namespace Aplib.Core.Belief.BeliefSets
 {
     /// <summary>
     /// This class can be inherited to define a set of beliefs for an agent.
-    /// All <i>public fields</i> defined in the inheriting class that implement <see cref="IBelief" /> 
+    /// All <i>public fields</i> defined in the inheriting class that implement <see cref="IBelief" />
     /// are automatically updated when calling <see cref="UpdateBeliefs" />.
     /// </summary>
-    public abstract class BeliefSet : IBeliefSet
+    public class BeliefSet : IBeliefSet
```

Comment:
Does this need to be abstract?

Reply:
No, that was the sonar comment. There is no abstract function.